.github/workflows/ci.yml

```
@@ -0,0 +1,103 @@
+          cache: true
+          cache-dependency-path: '**/go.sum'
+
+      - name: Check Go version
+        run: go version
+
+      - name: Run golangci-lint
+        uses: golangci/golangci-lint-action@v6
+        with:
+          version: v2.4.0
+          args: --timeout=5m
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)